File: src/CSharp.Repository/Services/HelloWorkRepository.cs
using CSharp.Repository.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharp.Repository.Services
{
    public class HelloWorkRepository : IHelloWorkRepository
    {
        public string SayHello => "Hello World";
    }
}

File: src/VB.Repository/Services/HelloWorkRepository.vb
Imports Microsoft.Extensions.DependencyInjection
Imports VB.Repository.VB.Repository.Interfaces

Namespace VB.Repository.Services

    Public Class HelloWorkRepository
        Implements IHelloWorkRepository

        Public ReadOnly Property SayHello As String Implements IHelloWorkRepository.SayHello
            Get
                Return "Hello World"
            End Get
        End Property

    End Class

End Namespace

